docs/zh/connectors/changelog/connector-hugegraph.md

```
@@ -0,0 +1,9 @@
+<details><summary> Change Log </summary>
+
+| Change | Commit | Version |
+| --- | --- |---------|
+|[Fix][Connector-V2] Enforce topology-safe execution order in HugeGraph multi-mapping sink|https://github.com/apache/seatunnel/commit/467980a6a0| dev |
```

markdownlint-cli2 (0.23.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "connector-v2"

(MD052, reference-links-images)